sonar/modules/documents/views.py

```diff
@@ -94,11 +96,36 @@ def detail(pid_value, view='global'):
 
         populate_files_properties(record)
 
+    # Resolve $ref properties
+    record = record.replace_refs()
+
     return render_template('documents/record.html',
                            pid=pid_value,
                            record=record)
 
 
+@blueprint.route('/projects/<pid_value>')
+def project_detail(pid_value, view='global'):
```

Comment:
Seems not the right approach. You should use `invenio_record_ui` configuration.

Reply:
I cannot do that because I listen to `view` parameter in the blueprint to store the organisation. If the view is registered in config, organisation is thus not stored.